src/pages/History/History.js

```diff
@@ -0,0 +1,24 @@
+import { Navbar, SideBar, LikedVideo } from "../../components";
+import { useLiked } from "../../context";
+
+export const History = () => {
+  const {
+    liked: { history }
```

Comment:
You can destructure this in the context itself while assigning it to value, so that you don't have to destructure it in every component

Reply:
updated, thanks ravi